swagauth/settings.py

```diff
@@ -152,15 +152,34 @@
 ]
 
 SWAGAUTH_SETTINGS = {
+    'bitbucket': {
+        'APP': {
+            'key': env.str('BITBUCKET_SECRET'),
```

Comment:
This must be called `client_id` instead of `key`.

Reply:
Edit #1. Or it's a right way to define bitbucket credentials?

Reply:
yes in bitbucket not client_id it has key

Reply:
Ok

Reply:
![image](https://user-images.githubusercontent.com/61553092/117975811-82e51300-b348-11eb-932f-186c1b5867c1.png)